scripts/_build/ec.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -z "$1" -o ! -e "$1" -o -z "$2" ]
+then
+    echo "$0 <config> <output>" >&2
+    exit 1
+fi
+
+source "$1"
+make -C ec clean
+make -C ec BOARD=${BOARD} VERSION=${VERSION} -j $(nproc)
+cp "$PWD/ec/build/${BOARD}/${VERSION}/ec.rom" "$2"
```

Comment:
What's the purpose of PWD here?

Reply:
It was failing earlier and I thought it was this. I'll remove it.